packages/components/src/components/route-instructions-step/route-instructions-step.tsx

```diff
@@ -410,6 +421,27 @@ export class RouteInstructionsStep implements ComponentInterface {
         </div>;
     }
 
+    /**
+     * Render the destination step.
+     *
+     * @returns {JSX.Element}
+     */
+    renderDestinationStep(): JSX.Element {
+        return <div class="step" onClick={e => this.stepClickHandler(e)}>
+            <div class="step__info destination">
+                <div class="step__destination">
+                    {this.isInternetExplorer ? null :
+                        <span class="step__action-icon step__action-icon--circled">
+                            <mi-icon icon-name="logo"></mi-icon>
+                        </span>
+                    }
+                    <div part="step-heading" class="step__heading">You have arrived</div>
```

Comment:
This string should also be part of the `DirectionsTranslations` interface and usage.

Reply:
It is part of the `DirectionsTranslations` interface, on file `directions-translations.interface.ts` on like 33 we have the `destination` string

Reply:
But `this.translationsData` is not used to write "You have arrived", so it is not translated.

Reply:
Oh, I see. How could we add the string so we could use it when displaying the `You have arrived step`? Where is the correct placement for it?

Reply:
It should be given as part of the `translations` prop, and then you can use it like:

```suggestion
                    <div part="step-heading" class="step__heading">{this.translationsData.arrived}</div>
```

Reply:
Added the string to the translations.